internal/dockerfile2dot/load_test.go

```diff
@@ -9,12 +9,16 @@ import (
 
 func TestLoadAndParseDockerfile(t *testing.T) {
 	type args struct {
-		inputFS afero.Fs
+		inputFS  afero.Fs
+		filename string
 	}
 
 	dockerfileFS := afero.NewMemMapFs()
 	_ = afero.WriteFile(dockerfileFS, "Dockerfile", []byte(`FROM scratch`), 0644)
 
+	dockerfile2FS := afero.NewMemMapFs()
+	_ = afero.WriteFile(dockerfile2FS, "Dockerfile2", []byte(`FROM scratch`), 0644)
+
```

Comment:
`dockerfile2FS` is currently not being used.

Reply:
It would be awesome if the tests could cover these 4 cases:

- default Dockerfile (not) found
- custom Dockerfile (not) found

The only one that's still missing is `custom Dockerfile found`, everything else already looks very good 👍

Reply:
Done. I also made Dockerfile2 different, so that it's clear that they're not mixed up.